src/plugins/ui_stateful/selection.ts

```diff
@@ -582,6 +582,14 @@ export class GridSelectionPlugin extends UIPlugin {
       currentIndex += 1;
     }
 
+    this.dispatch("MOVE_RANGES", {
```

Comment:
Sure but ... isn't MOVE_RANGES already dispatched inside the cut/paste ? Do you know why it isn't working ? Is cut/paste also broken in the grid ?

Reply:
> isn't MOVE_RANGES already dispatched inside the cut/paste ?
Not since the 17.4 refactor

Ty for the review, I indeed focused on solving the problem at hand and didn't see that the clipboard only copied the content of the cells and not the rest :/